Assets/Scripts/Trackers/PatrollingTracker.cs

```diff
@@ -45,7 +45,7 @@ public class PatrollingTracker : Tracker<PatrollingVisualizer, IPatrollingVisual
 
         //TODO: TotalCycles is not set any where in the code
         public int TotalCycles { get; }
-        public bool HaveToggledSecondStoppingCriteria { get; set; }
+        public bool HaveToggledSecondStoppingCriteria { get; }
```

Comment:
This can be completely removed.
In the PatrollingSimulation change in line 51 PatrollingTracker.HaveToggledSecondStoppingCriteria to _scenario.StopAfterDiff